book/src/setup.md

```diff
@@ -17,9 +17,6 @@ The additional requirements for developers are:
   some dependencies. See [`Installation Guide`](./installation.md) for more info.
 - [`java 17 runtime`](https://openjdk.java.net/projects/jdk/). 17 is the minimum,
   used by web3signer_tests.
-- [`libpq-dev`](https://www.postgresql.org/docs/devel/libpq.html). Also known as
-  `libpq-devel` on some systems.
-- [`docker`](https://www.docker.com/). Some tests need docker installed and **running**.
```

Comment:
I believe removing `watch` also removes these compilation dependencies, would someone be able to confirm this?

Reply:
Yeah, agree. We don't use Docker for any of the other tests.